.github/workflows/publish_docs_to_wiki.yml

```diff
@@ -2,40 +2,18 @@ name: Publish docs to Wiki
 
 on:
   push:
-    paths:
-      - docs/**
```

Comment:
@manh-t I could be wrong, but instead of removing this, should it be `- .github/wiki/**` instead?

Same as iOS has done: https://github.com/nimblehq/ios-templates/blob/master/.github/workflows/publish_docs_to_wiki.yml

Reply:
@manh-t Without `paths:` the workflow will still work correctly, indeed 👍 

But from my understanding, keeping it will **optimize** our CI:
- If we remove `paths:`,  this workflow will be called every time we push to main
- If we keep `paths:`,  this workflow will only be called if something has been updated inside `.github/wiki/**`
![Screenshot 2022-10-28 at 09 18 03](https://user-images.githubusercontent.com/18277915/198463730-047e1d67-18c5-48b9-8f59-756bbcb346a5.png)

**Reference**: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths

Reply:
@Tuubz valid point 👍 Let me bring it back so we can limit the scope of running this workflow

Reply:
@Tuubz updated at 45d336e559aef785136f41efa8f0de84bc254dc9